src/main/scala/com/microsoft/hyperspace/actions/RefreshActionBase.scala

```diff
@@ -71,8 +71,10 @@ private[actions] abstract class RefreshActionBase(
   // Reconstruct a df from schema
   protected lazy val df = {
     val relations = previousIndexLogEntry.relations
-    val latestRelation =
-      Hyperspace.getContext(spark).sourceProviderManager.refreshRelationMetadata(relations.head)
+    val latestRelation = Hyperspace.getContext(spark)
+      .sourceProviderManager
```

Comment:
nit: doesn't scalafmt give you
```suggestion
    val latestRelation = Hyperspace
      .getContext(spark)
      .sourceProviderManager
```
?

Reply:
Done